<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>yaflight4w</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="gboxAircraft">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>Aircraft</string>
    </property>
    <widget class="QPushButton" name="pbtLaunch">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Launch!</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cboAircrafts">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblAircraftPreview">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>181</width>
       <height>101</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gboxOptions">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>171</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtExit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtAbout">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>About</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtAddModel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>90</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>New Model</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtSaveConf">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save conf</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtLoadConf">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>150</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Load conf</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>20</width>
      <height>271</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txaLog">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>371</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
